template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
```

Comment:
This looks like the script from HTTP with the proof_marker modification, are there other changes to notice.

Reply:
No other changes, but kissat is not included in this PR. As discussed previously, we'll leave that to a future PR.